controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentTriggerTest.java

```diff
@@ -1216,4 +1217,42 @@ public void testEagerTests() {
         app.assertNotRunning(stagingTest);
     }
 
+    @Test
+    public void test() {
```

Comment:
I think there's a better name to be found.

Reply:
😂

Reply:
Had no idea what the problem was when I started writing the test ...

Reply:
```suggestion
    public void testTriggeringOfIdleTestJobsWhenFirstDeploymentIsOnNewerVersionThanChange() {
```

Reply:
Sounds familiar. 🙂